orderapi/src/test/java/fi/hel/verkkokauppa/order/service/order/OrderServiceTest.java

```diff
@@ -78,32 +85,32 @@ class OrderServiceTest extends TestUtils {
     @Autowired
     private SubscriptionController subscriptionController;
 
-    @Autowired
```

Comment:
All fields that are removed here is because they were not used at all in the tests.